vllm/model_executor/model_loader.py

```diff
@@ -116,7 +116,7 @@ def get_model(model_config: ModelConfig, device_config: DeviceConfig,
                 # to retain tensorizer args from CLI.
                 model_config.load_format = ParameterizedLoadFormat(
                     model_config.load_format)
-                model_config.load_format.params = (
+                model_config.load_format.params = (  # type: ignore
```

Comment:
@sangstar I found this approach is a bit hacky (dynamically loading load_format) and doesn't work well with typing. Is there any good suggestion to fix it?

Reply:
I believe @Yard1 's refactor addressed this :D